simpleclient_dropwizard/src/main/java/io/prometheus/client/dropwizard/DropwizardExports.java

```diff
@@ -20,15 +20,29 @@
 public class DropwizardExports extends io.prometheus.client.Collector implements io.prometheus.client.Collector.Describable {
     private static final Logger LOGGER = Logger.getLogger(DropwizardExports.class.getName());
     private MetricRegistry registry;
+    private MetricFilter metricFilter;
     private SampleBuilder sampleBuilder;
 
     /**
-     * Creates a new DropwizardExports with a {@link DefaultSampleBuilder}.
+     * Creates a new DropwizardExports with a {@link DefaultSampleBuilder} and {@link MetricFilter#ALL}
      *
      * @param registry a metric registry to export in prometheus.
      */
     public DropwizardExports(MetricRegistry registry) {
         this.registry = registry;
+        this.metricFilter = MetricFilter.ALL;
+        this.sampleBuilder = new DefaultSampleBuilder();
+    }
+
+    /**
+     * Creates a new DropwizardExports with a {@link DefaultSampleBuilder} and custom MetricFilter
```

Comment:
Full stop for consistency, and elsewhere.

Reply:
Fixed